content/news/2024-12-12-mixxing-up-pen-and-paper.md

```diff
@@ -0,0 +1,122 @@
+title: Mixxxing up Role Playing
+authors: betalars
+status: draft
+tags: user storys
+comments: yes
+
+
+> It is cold and moist. You are deep within the belly of a huge machine surrounded by overgrowth and broken glass. You know she wants to kill you. After all: you did the same to her. She wants revenge and the only thing keeping you alive is her unstoppable desire to test.
+>
+> The scratching of plates and distant screams of broken turrets surround you. You are alone; observed by an all-seeing machine. Your only chance of survival is to comply.
+>
+> So, testing. Right.
+>
+> You enter the next chamber.
+>
+> "Did you know that people with a bad conscience get more easily startled by loud nois- " *CHOOOOOOOO*
+>
+> *She is out to kill you.*
+> 
+> The harsh red light of a laser illuminates the huge test chamber, when suddenly, a cube drops from the ceiling. But it is not like the others. Every Cube so far had an Aperture printed on it. But this one ... a heart? Memories of betraying an old friend flash before your eyes.
+>
+> You get closer, carefully stepping over the deadly laser beam, when you hear a faint sound. It is friendly. It does not want to hurt you.
+>
+> You go closer to the cube.  
+> Is that voice coming from the ... cube?  
+> You crouch next to it.
+>
+> It hums. A small melody. Fighting against the eary noise around you.
+
+If I had to pick a favorite moment in a video game, it would be this: a perfect example of how mixing and dynamic soundtracks can not only enhance, but tell a story outright.
+So let's use this to spice up pen and paper.
+
+*[Play "Love as a construct" from Portal 2 on YouTube Music.](https://music.youtube.com/watch?v=UNhBzbN2rnA)*
+
```

Comment:
```suggestion

*As music is better listened to than explained, I curated some songs for you. I use YouTube not because it is my favorite platform, but because it likely is the most accessible.*

```